namespace/namespace_commands.go

```diff
@@ -303,7 +303,7 @@ func DeleteNamespace(c *cli.Context) error {
 
 	promptMsg := color.Red(c, "Are you sure you want to delete namespace %s? Type namespace name to confirm:", ns)
 	if !common.Prompt(promptMsg, c.Bool(common.FlagYes), ns) {
```

Comment:
We shouldn't print the prompt if the user wants to auto-confirm.

Reply:
We print both the prompt and response, which I think is okay for this hotfix. I don't really want to expand the scope of this PR since we're working on a rewrite.